templates/Dockerfile-template.alpine

```diff
@@ -18,44 +18,46 @@ ARG        LD_LIBRARY_PATH=/opt/ffmpeg/lib
 ARG        PREFIX=/opt/ffmpeg
 ARG        MAKEFLAGS="-j2"
 
-ENV         %%ENV%%
-
-RUN     buildDeps="autoconf \
+RUN set -xe && \
+        buildDeps="autoconf \
                    automake \
                    bash \
                    binutils \
                    bzip2 \
                    cmake \
-                   curl \
                    coreutils \
+                   curl \
                    diffutils \
+                   expat-dev \
                    file \
                    g++ \
                    gcc \
+                   gnutls-dev \
                    gperf \
                    libtool \
                    make \
-                   python \
                    openssl-dev \
+                   python \
                    tar \
+                   xz \
                    yasm \
-                   nasm \
-                   zlib-dev \
-                   expat-dev" && \
-        apk  add --no-cache --update ${buildDeps}
+                   zlib-dev" && \
+        apk add --no-cache --update ${buildDeps}
 %%RUN%%
 
-RUN \
+RUN set -xe && \
+    rm -f /usr/local/bin/nasm /usr/local/bin/ndisasm && \
     ldd ${PREFIX}/bin/ffmpeg | grep opt/ffmpeg | cut -d ' ' -f 3 | xargs -i cp {} /usr/local/lib/ && \
     cp ${PREFIX}/bin/* /usr/local/bin/ && \
     cp -r ${PREFIX}/share/ffmpeg /usr/local/share/ && \
     LD_LIBRARY_PATH=/usr/local/lib ffmpeg -buildconf
 
 ### Release Stage
 FROM        base AS release
-MAINTAINER  Julien Rottenberg <julien@rottenberg.info>
+LABEL maintainer="Julien Rottenberg <julien@rottenberg.info>"
+LABEL ffmpeg=true
```

Comment:
would that make sense to put :
```
ffmpeg_version=xyz
ffmpeg_enabled=true
```
as label ?

Reply:
ok